LICENCE.md

```diff
@@ -0,0 +1,19 @@
+Copyright (c) <year> <copyright holders>
```

Comment:
you need to change here `<year>` and `<copyright holders>` ;)

Reply:
ok it's good